fs/params/validate_model/age_categories.csv

```diff
@@ -0,0 +1,9 @@
+age
```

Comment:
Can you put these out of a csv but rather a config.py and import them natively? These are fixed values so we don't really need to import them from csvs. By the look of it they are no that complex, you can even hard code them in the program